### DISCUSSION TITLE
helmwave.yml for goldpinger

### DISCUSSION THREAD

**Discussion by patsevanton** [helmwave/helmwave#836] [Q&A]
Hello! Thanks for helmwave
I try generate helmwave.yml for https://artifacthub.io/packages/helm/okgolove/goldpinger

I have 2 command
```
helm repo add okgolove https://okgolove.github.io/helm-charts
helm install my-goldpinger okgolove/goldpinger --version 6.1.2
```

How generate helmwave.yml ?

**Comment by zhilyaev** [comment]
no way

**Comment by zhilyaev** [comment]
You can watch the [video](https://www.youtube.com/watch?v=zZoY0TTvyjs) for an explanation for russian-speaking users
